trailingstoploss_tp.py

```diff
@@ -213,29 +243,37 @@ def process_deals(thebot, section_config):
             deal_id = deal["id"]
 
             if deal["strategy"] in ("short", "long"):
-                current_deals.append(deal_id)
+                # Check whether the actual_profit_percentage can be obtained from the deal,
+                # If it can't, skip this deal.
+                try:
+                    current_deals.append(deal_id)
 
```

Comment:
Persoonlijk vind ik een try/catch niet zo heel mooi hiervoor, des te meer omdat er ook een andere value ongeldig kan zijn en in de catch rapporteren we dan dat het altijd de profit is die ongeldig is...maar dat hoeft dus niet!

Ik had zelf in gedachten om geen try/catch te gebruiken, maar waar je nu de try hebt staan een if toe te voegen:

if deal["actual_profit_percentage"] is "":
   <logmelding met dat de actual_profit_percentage voor deal x ongeldig is>
   continue;
   
Dan is specifiek duidelijk op welke fout we controleren, en wat er dan fout is. Andere checks kunnen dan later op dezelfde manier toegevoegd worden.